kuma/users/jinja2/users/includes/user_title_info.html

```diff
@@ -70,4 +70,38 @@ <h1 class="user-title">
   </ul>
 </div>
 {% endif %}
-<div class="user-since">{% trans date_joined=datetimeformat(detail_user.date_joined, format='date') %}Member since {{ date_joined }}{% endtrans %}</div>
\ No newline at end of file
+
+<ul class="user-links">
+  {% if detail_user.twitter_url %}
+    <li class="twitter"><i class="icon-twitter" aria-hidden="true"></i><a href="{{ detail_user.twitter_url }}" rel="me external" class="url">{{ _("Twitter") }}</a></li>
+  {% endif %}
+
+  {% if accounts.github %}
+    {% for github_account in accounts.github %}{% if github_account.extra_data %}
+    <li class="github"><i class="icon-github" aria-hidden="true"></i><a href="{{ github_account.get_profile_url() }}" rel="me external" class="url">{{ _("GitHub") }}</a></li>
```

Comment:
`aria-hidden` is on the icons, they should always be hidden.

I don't have aria on the list item that is being hidden or shown partially because I didn't know how to hook aria-controls into the the jquery UI widget which is the trigger for showing/hiding this field and partially because I think the logical format of the page is fine from a screen reader perspective. If they have added interests in the preceding step that reveal the expertise options then they will encounter the expertise options naturally when they move to the next step in the form. If something goes wrong and they don't, that's okay, the field isn't required.